docs/configuration/configuration.md

```diff
@@ -2797,6 +2797,12 @@ write_relabel_configs:
 # For the `io.prometheus.write.v2.Request` message, this option is noop (always true).
 [ send_native_histograms: <boolean> | default = false ]
 
+# When enabled, remote-write will resolve the URL host name via DNS, choose one of the IP addresses at random, and connect to it. 
+# When disabled, remote-write relies on Go's standard behavior, which is to try to connect to each address in turn.
```

Comment:
Maybe add "in a fixed order" to emphasize the difference this option actually makes?

Reply:
Suspect you mean to emphasise the order is fixed by the client not the server.